.github/workflows/build.yml

```diff
@@ -40,6 +40,7 @@ jobs:
         run: |
           python -m pip install --upgrade pip wheel
           pip install -e .
+          pip install -e .[test,websurfer]
```

Comment:
Move this line to contrib-tests.yml because we need to make sure the build succeeds without optional dependency.

Reply:
Thanks. Will do.

I also really appreciate your early feedback. I know there are a lot of necessary bits that are currently missing (tests etc.), and will add them ASAP before transitioning out of Draft.